src/components/Avatar/Avatar.tsx

```diff
@@ -48,33 +47,33 @@ export interface AvatarProps extends VibeComponentProps {
 
 const Avatar: React.FC<AvatarProps> = ({
   id,
-  type,
+  type = AvatarTypes.TEXT,
   className,
-  textClassName,
-  size,
+  textClassName = "",
+  size = AvatarSizes.LARGE,
   src,
   icon,
   text,
   tooltipProps,
   ariaLabel,
-  withoutTooltip,
+  withoutTooltip = false,
   role,
-  backgroundColor,
+  backgroundColor = elementColorsNames.CHILI_BLUE,
```

Comment:
Why is this not capitalized?

Reply:
old code, was like this before, not under this pr scope